src/sdk/namespace_fs.js

```diff
@@ -406,17 +466,22 @@ class NamespaceFS {
     /////////////////
 
     async read_object_md(params, object_sdk) {
+        let target_file;
+        const fs_account_config = this.set_cur_fs_account_config(object_sdk);
         try {
-            const fs_account_config = this.set_cur_fs_account_config(object_sdk);
             await this._load_bucket(params, fs_account_config);
             const file_path = this._get_file_path(params);
             const stat = await nb_native().fs.stat(fs_account_config, file_path);
-            dbg.log0(file_path, stat);
             if (isDirectory(stat)) throw Object.assign(new Error('NoSuchKey'), { code: 'ENOENT' });
-            return this._get_object_info(params.bucket, params.key, stat);
+            // check_md_conditions(params.md_conditions, stat);
+            target_file = await nb_native().fs.open(fs_account_config, file_path, undefined, get_umasked_mode(config.BASE_MODE_FILE));
+            const xattr = await target_file.getxattr(fs_account_config);
```

Comment:
Name it fs_xattr so we can differentiate the two different types of xattr object we use (fs / s3)
```suggestion
            const fs_xattr = await target_file.getxattr(fs_account_config);
```